x-pack/plugins/fleet/server/types/models/agent.ts

```diff
@@ -22,7 +22,6 @@ export const AgentTypeSchema = schema.oneOf([
 export const NewAgentActionSchema = schema.oneOf([
   schema.object({
     type: schema.oneOf([
-      schema.literal('POLICY_CHANGE'),
       schema.literal('UNENROLL'),
       schema.literal('UPGRADE'),
       schema.literal('POLICY_REASSIGN'),
```

Comment:
Further investigation needs to be done, but I'm not sure we use these other actions from the UI either (we have dedicated APIs for this). We may want to consider not allowing these others from the API layer either.